tests/system/aiplatform/test_dataset.py

```diff
@@ -302,7 +307,7 @@ def test_create_tabular_dataset_from_dataframe(self, bigquery_dataset):
                 == bigquery.SchemaField(name="datetime_col", field_type="DATETIME")
                 if re.match(
                     r"3.*",
-                    pkg_resources.get_distribution("google-cloud-bigquery").version,
+                    metadata.version("google-cloud-bigquery"),
```

Comment:
Let's use `__version__` instead of `importlib.metadata`. `google.cloud.bigquery.__version__`